index.d.ts

```diff
@@ -75,15 +74,16 @@ declare module "react-native-calendar-strip" {
       scrollable?: boolean;
       startingDate?: Date;
       selectedDate?: Date;
-      onDateSelected?: (date: Date) => void;
-      onWeekChanged?: (start: Date, end: Date) => void;
-      onHeaderSelected?: (dates: {weekStartDate: Date, weekEndDate: Date}) => void;
+      onDateSelected?: ((date: Moment) => void);
+      onWeekChanged?: ((start: Moment, end: Moment) => void);
+      onHeaderSelected?: (({weekStartDate: Moment, weekEndDate: Moment}) => void);
```

Comment:
Let's change only the line like below.  

```tsx
onHeaderSelected?: ((dates: {weekStartDate: Moment, weekEndDate: Moment}) => void);
```

It fixes TypeScript linting-errors.

Reply:
I've updated the PR with your feedback. I've also change the other `Date` to `Moment`, I was having an error when I was passing a Date for `datesWhitelist` with the function in CalendarDay.js `date.isBetween()` and `date.isSame()`